clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -326,6 +330,10 @@ private Exception invokePartitionsRevoked(final Set<TopicPartition> revokedParti
 
     private Exception invokePartitionsLost(final Set<TopicPartition> lostPartitions) {
         log.info("Lost previously assigned partitions {}", Utils.join(lostPartitions, ", "));
+        Set<TopicPartition> lostPausedPartitions = subscriptions.pausedPartitions();
+        lostPausedPartitions.retainAll(lostPartitions);
+        if (!lostPausedPartitions.isEmpty())
+            log.info("The pause flag in partitions [{}] will be removed due to lost.", Utils.join(lostPausedPartitions, ", "));
```

Comment:
nit: due to **partition** lost

Reply:
Thank you for your rigorous attitude.
Forgive my bad English.